### DISCUSSION TITLE
does upload_translations override translations, or just update new ones?

### DISCUSSION THREAD

**Discussion by malytomas** [crowdin/github-action#233] [Q&A] [answered]
When I create new keys, I also write english translation (which is also the source language). I want to upload it to crowdin, so that others can improve it (I am not native english speaker). Will this override the translation in the crowdin, or will the crowdin version have higher priority?

**Comment by andrii-bodnar** [comment, marked as answer]
@malytomas the new keys (source strings) will be uploaded to Crowdin and the existing ones will be updated. You can also use the [`update_option` configuration](https://developer.crowdin.com/configuration-file/#changed-strings-update) to specify whether to preserve the existing translations and approvals for the updated strings or not.

**Reply by malytomas** [reply]
thanks ;)
